src/main/proto/netflix/titus/agent.proto

```diff
@@ -287,6 +289,9 @@ message ContainerInfo {
 
     /// (Optional) Amount of shared memory to allocate (must be <= memoryMB)
     optional uint32 shmSizeMB = 40;
+
+    /// (Optional) IP address allocation for the task
+    optional com.netflix.titus.SignedAddressAllocation signedAddressAllocation = 41;
```

Comment:
Do we need qualified name here or `SignedAddressAllocation` would be enough?

Reply:
My impression is that since `agent.proto` uses `package messages` it needs to be qualified to access the `titus_base.proto`'s `package com.netflix.titus`. I'm not able to get it to build without the package qualification.

Reply:
I have missed that it is proto 2. We are mixing now two protobuf versions.

Reply:
Yes, though that should be fine.

Reply:
@tbak , let me know if there is a specific concern here.

Reply:
No concern. We do not use qualified names in other places in proto3 IDLs. But if proto2 requires that, lets do it.

Reply:
:+1: